### ISSUE TITLE
Accessibility: Narrator doesn't announce collapsed/expanded input states

### ISSUE THREAD

**Issue by DonniewGit** [OfficeDev/office-ui-fabric-react#7300]
#### Environment Information 
 
Application Name :Power Platform Environments List Application  
Operating System: Windows 10
Browser Version Edge42 

#### Please provide a reproduction of the bug in a codepen
 
https://codepen.io/anon/pen/JeqKZm?&editable=true#0
https://codepen.io/anon/pen/eQazjV?&editable=true#0
 
#### Actual behavior
 
Narrator does not announce expanded or collapsed state of Dropdown/Combobox component.
 
#### Expected behavior
 
Screen reader should announced the state of the combobox and dropdown after getting expanded and collapsed.

#### Priorities and help requested
 
Are you willing to submit a PR to fix? No
 
Requested priority: Normal
 
Products/sites affected: Power Platform Admin Center

**Comment by kkjeer** [comment]
Thank you for opening this issue! This is currently being tracked as Bug 2 in #6326.
